core/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -127,6 +128,16 @@
     public static final Setting<Integer> MAX_SLICES_PER_SCROLL = Setting.intSetting("index.max_slices_per_scroll",
         1024, 1, Property.Dynamic, Property.IndexScope);
 
+    /**
+     * The number of shard copies required to be active before returning from an index creation operation.
+     */
+    public static final Setting<ActiveShardCount> WAIT_FOR_ACTIVE_SHARDS_SETTING =
```

Comment:
@bleskes This parameter serves as a default if wait_for_active_shards is not specified in the request.  We could just use the setting and it would simplify things.  I think originally I was trying to keep consistent with the way indexing requests enable specifying the write consistency param, but upon reflection, it does seem unnecessary here.  Would you like me to remove it?